Yosemite/YosemiteTests/Mocks/CardPresentPayments/MockCardReaderService.swift

```diff
@@ -41,10 +43,18 @@ final class MockCardReaderService: CardReaderService {
     func start(_ configProvider: CardReaderConfigProvider) {
         didHitStart = true
         didReceiveAConfigurationProvider = true
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {[weak self] in
```

Comment:
Is this required or just for testing, a comment will help to new-comers like me!